packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php

```diff
@@ -17,7 +17,10 @@
                 {!! view_render_event('bagisto.shop.categories.toolbar.filter.before') !!}
 
                 <!-- Product Sorting Filters -->
-                <x-shop::dropdown position="bottom-left">
+                <x-shop::dropdown 
+                        class="z-[1]" 
+                        position="bottom-left"
+                        >
```

Comment:
To improve readability fix the extra space

Reply:
is it okay now ? @shivendra-webkul

Reply:
Remove the space before the ">" in the tag.

It should be something like this 

Reference :

<x-shop::dropdown 
    class="z-[1]" 
    position="bottom-left"
>

Reply:
@hussaincoder23, Thank you for your contribution.